cc/game.h

```diff
@@ -95,6 +97,9 @@ class Game {
 
   void UndoMove();
 
+  // Marks the most recent move as trainable.
+  void SaveLastMove();
```

Comment:
`SaveLastMove` could mean a bunch of different things. How about something along the lines of `MarkLastMoveAsTrainable`?

Reply:
Done.